ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json

```diff
@@ -69,6 +69,34 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
   },
```

Comment:
@samuelzerozero, sorry I can't write on Slack at the moment (ZScaler).
I saw your message, though. Talk to Tom. This is the first time I've ever seen ZScaler.

Reply:
The review should be ready for you, by the way.
Sorry for its size.

Reply:
@jvictorcf some comments in the story.